pkg/tlsf/.gitattributes

```diff
@@ -1,2 +0,0 @@
-# Ignore whitespace errors inside these patch files by regarding them as binary.
-patch.txt binary
```

Comment:
That's why I introduced a separate repository for TLSF (and would strongly advice to have this for every package that requires patches). Spamming the output of `git diff` with rather useless output seems undesirable to me.